Desktop version for "My Portfolio" application.

lmunozm1702 opened the pull request:
Include the 4 sections and the "navbar" developed previously for mobile devices, in a version for desktop devices:

>> Main Content
>> Works
>> About me
>> Contact Form

As requested the @media queries are defined on min-width: 768px.

The font sizes for both versions (mobile and desktop) were specified in "vw" units transforming the "px" sizes from FIGMA. That's to improve responsiveness.

The contents in previous versions without a container were included in a "flex" container to make easy the display on different devices.

lmunozm1702 committed:
Css organizing

Update index.html and styles.css to organize the code.

lmunozm1702 committed:
Navbar and Main Contente

Update style.css and index.html to include @media behaivor for navbar and main content sections.

Highlights (when min-width:768px) :
- Text menu visible and hamburguer menu hidden.
- Change of font sizes when indicated on Figma.
- Change Margins of the sections.
- Include links and anchors to sections from menu.
- Change backgrounds images on main section.

Add header-bg.png and logo-placeholder to /assets/images folder.

lmunozm1702 committed:
Work cards

Update index.html and styles.css to be responsive on @media only screen and (min-width: 768px)

Highlights:
- Change layout of the contents to be included in Flex containers to improve responsiveness (work-main and work-item)
- Alignment of the card image to be at left or right  interleaved.
- Review and change of all fonts (size, width and inteline)
- Change images takes from Figma.
- Implement hide/show to categories (shows "Ruby on rails" than its hide in mobile)

Add facebook-360.png, multi-post.png, tonic.png and uber-navigation.png
Delete my_logo.png, project-multi-post-2.png,  project-multi-post.png, project-tonic-png and project-tonic-2.png
From /assets/images folder.

lmunozm1702 committed:
Expereince section

Update index.html and styles.css to:

- Include the "Experience" section using responsive layout for @media only screen and (min-width: 768px)

- Change the font sizes and horizontal margins and padding from px to vw (the figma layout it's for 1440 px with and the requirement has for 768 px) I calculate the vw from 1400px template using https://www.quick-tools.net/16px-to-vw-1440vp

lmunozm1702 committed:
"About me" and "Contact form" sections

Update style.css and index.html to include both sections.

Add contact-form-background-shape-desktop.png into /assets/images folder

include fonts an border referenced in "vw" units to improve responsiveness (just into @media only screen and (min-width: 768px))

lmunozm1702 committed:
Font responsives in mobile version

Update index.html and styles.css to change the units of the fonts in the mobile version from px to VW.

The conversion took the width (375px) and font sizes from FIGMA.

Add the black-line.png file into /assets/images folder.

lmunozm1702 committed:
Fix Stylelint linter issues

Update styles.css to fix the issues:

assets/css/style.css
 166:3  ✖  Unexpected unknown type selector "nav-right-text"   selector-type-no-unknown
 461:3  ✖  Expected empty line before rule                     rule-empty-line-before
 613:3  ✖  Expected empty line before rule                     rule-empty-line-before

Error: Process completed with exit code 2.

sajid-munawar commented on assets/css/style.css:
```diff
@@ -588,6 +588,12 @@ label {
+
 @media only screen and (min-width: 768px) {
   .contact-me {
     padding: 114px 22.222vw 8px 22.222vw;
```
- [x] Please kindly make sure you have added the exact background as it is the core requirement and also makes your section much more attractive.
## your design👇
![formstbackdesk](https://user-images.githubusercontent.com/66992901/193128885-11c09a25-883b-433c-92fc-2ac2487491a5.PNG)

## Figma design👇
![formbackdesk](https://user-images.githubusercontent.com/66992901/193128829-2dfe68ce-080f-46b5-b1af-43a761c9f8ee.PNG)

sajid-munawar commented on assets/css/style.css:
```diff
@@ -729,8 +735,8 @@ label {
 .contact-me-submit {
   color: #6070ff;
   padding: 12px 16px 12px 16px;
-  font-size: 17px;
-  line-height: 20px;
+  font-size: 4.533vw;
+  line-height: 1.176;
   border-radius: 8px;
   border: none;
   margin-bottom: 11px;
```
- [x] Please decrease the size of your button, make it as the figma design. now your page look messy with that large button. kindly fix it.
## your design 👇
![formstbtn](https://user-images.githubusercontent.com/66992901/193129458-6290481b-90d4-4c7f-9ff0-6b9ba66db24c.PNG)

## Figma design👇
![formbtn](https://user-images.githubusercontent.com/66992901/193129540-4afc8b8a-c37f-4334-91f4-e0c68924d364.PNG)

sajid-munawar commented on assets/css/style.css:
```diff
@@ -213,8 +211,8 @@ label {
   font-family: "Poppins", sans-serif;
   font-style: normal;
```
- [ ] Please make sure your background for the `main-content` section is displayed. kindly check

## your design👇
![deskbackst](https://user-images.githubusercontent.com/66992901/193130714-2e6bda18-692f-4b69-aae7-200972d997a4.PNG)

## Figma design👇
![deskback](https://user-images.githubusercontent.com/66992901/193130789-7435ae57-05b1-4a1b-bd2c-cb2324979869.PNG)

sajid-munawar commented on assets/css/style.css:
```diff
@@ -162,6 +159,7 @@ label {
 
 .logo-placeholder img {
   margin: 10px 12px 10px 12px;
+  width: 100%;
 }
 
```
- [x] Please make sure you stick to the figma design. kindly arrange the padding and margins accordingly. and also try to apply the same fonts and typography.
## your design👇
![facest](https://user-images.githubusercontent.com/66992901/193131516-c50cf2d1-a39d-495d-93e6-f5c2ea4b2f7c.PNG)

## Figma design👇
![facefigma](https://user-images.githubusercontent.com/66992901/193131567-c642119b-a370-4784-bceb-868cf9764c88.PNG)
#### Please apply for all your cards😊.

lmunozm1702 replied:
Hi @sajid-munawar it's done the second correction!!

sajid-munawar commented on assets/css/style.css:
```diff
@@ -123,7 +120,7 @@ label {
     font-style: normal;
     font-weight: 500;
     font-size: 1.042vw;
-    line-height: 1.389vw;
+    line-height: 1.389;
     color: #344563;
```
- [ ] Please align your text center as the Figma design.
## your design 👇
![langstdesk](https://user-images.githubusercontent.com/66992901/193132542-444790c6-48e2-48dd-ae03-6d557ac88cac.PNG)

## Figma design👇
![langfigmadesk](https://user-images.githubusercontent.com/66992901/193132641-d522cd37-8040-4539-b81e-7f4643dca2a9.PNG)

sajid-munawar reviewed (changes requested):
# Changes Required 
 ### Hi @lmunozm1702 
 Nice job, you are almost there.
 
 ![](https://camo.githubusercontent.com/34482800e1b92cb79ddca06edb4051edc407cb190f896e1e949e596aa9073851/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
 
 ## (Highlights) Good Points: 
 * Good commit messages. :heavy_check_mark:
 * No linter errors. :heavy_check_mark:
 * Followed Github flow :heavy_check_mark:
 * Professional README file. :heavy_check_mark:
 * PR has a good title and summary :heavy_check_mark:

 ## (Changes Required) Aspects to improve: 
 * [ ]  Please check my inline comments.
 
 ## [Optional] suggestions:
 * Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 **Cheers and Happy coding! :clap::clap::clap:**
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear
 Please, remember to tag @sajid-munawar in your question so I can receive the notification.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

lmunozm1702 committed:
Code Review issues

Update index.html and styles.css to solve the issues:

- Background images not displayed: I change the name of the images to refresh in my desktop the images are displayed correctly.

- Work Cards: solved with flex-basis property.

- Font in button "Get in touch" solved.

- Centering "Languages" solved reducing gap property.

lmunozm1702 commented:
HI @sajid-munawar, all issues solved.

sajid-munawar commented on index.html:
```diff
@@ -192,7 +192,7 @@ <h2>About Myself</h2>
                 <div class="code-experience">
                     <div class="list-header">
                         <ul class="experience-nav">
-                            <li>Languajes</li>
+                            <li>Languages</li>
                             <li><i class="fa-solid fa-angle-down"></i></i></li>
                         </ul>
```
- [x] Nice implementation of previous review. please stick to the Figma and control overlaping while changing from desktop version to the mobile version give it a margin and padding left.
## your design👇
![overlap](https://user-images.githubusercontent.com/66992901/193147117-08450d3c-c9bf-425b-aa16-8da925c76c98.PNG)

lmunozm1702 replied:
Hi @sajid-munawar it's done!

sajid-munawar reviewed (changes requested):
Hi @lmunozm1702 , 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.
  - [ ] As the previous reviewer mentioned please add these background images to your portfolio as the Figma design👇
  
![firstfigma](https://user-images.githubusercontent.com/66992901/193147819-a430c1d9-c856-46de-90ef-c10c17511d12.PNG)
![lastfigma](https://user-images.githubusercontent.com/66992901/193147861-d84605f1-28b1-467d-8e70-e717c9bbaffb.PNG)
 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

lmunozm1702 committed:
Code Review 2: issue solved

Update index.html and style.css to solve the issue with the text at rigth of the image on Languages section.

I align too the button "Get in touch" to left for mobile devices.

lmunozm1702 committed:
Solve Stylelint issue

Update styles.css to solve:

assets/css/style.css
 584:3  ✖  Expected empty line before rule   rule-empty-line-before

Error: Process completed with exit code 2.

jerryowusu commented on index.html:
```diff
@@ -60,223 +73,256 @@ <h3>Tonic</h3>
+                </ul>
+                <div><button class="see-project-button">Get my resume</button></div>
+            </main>
+            <aside class="about-myself-aside">
+                <div class="code-experience">
+                    <div class="list-header">
+                        <ul class="experience-nav">
+                            <li>Languages</li>
+                            <li><i class="fa-solid fa-angle-down"></i></i></li>
+                        </ul>
+                    </div>
+                    <div>
+                        <ul class="experience-list">
+                            <li class="experience-item">
+                                <div class="experience-img">
+                                    <img src="assets/images/javascript.png" alt="javaScript logo">
+                                </div>
+                                <div>
+                                    <p class="languaje-item">JavaScript</p>
+                                </div>
+                            </li>
+                            <li class="experience-item">
+                                <div class="experience-img">
+                                    <img src="assets/images/html5.png" alt="HTML-5 logo">
+                                </div>
+                                <div>
+                                    <p class="languaje-item">HTML</p>
+                                </div>
+                            </li>
+                            <li class="experience-item">
+                                <div class="experience-img">
+                                    <img src="assets/images/css.png" alt="CSS logo">
+                                </div>
+                                <div>
+                                    <p class="languaje-item">CSS</p>
+                                </div>
+                            </li>
+                        </ul>
+                    </div>
```
[OPTIONAL] From the figma design, you can see that it is only the language sections which has been extended or open to show the languages available whiles the other two parts (Frameworks and Skills) are closed. I can see from your design that you opened those two and added some icons. You also changed the arrows on the right as shown in the image below. It is good to be innovative and creative, :man_artist:  but the purpose of this exercise is to conform to the dimensions of the figma design as strictly as possible. :disappointed_relieved:  You will get more opportunities to be creative in subsequent projects, but for now Kindly revert this changes back to the original as depicted on the figma designs. :pleading_face: 

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-29 23-07-17](https://user-images.githubusercontent.com/65233860/193158449-e2baae57-7441-4c09-9a1f-20b008cdc1e4.png)  |  ![Screenshot from 2022-09-29 23-07-43](https://user-images.githubusercontent.com/65233860/193158481-e98fd072-b1e5-4871-aa0e-19226a03d48e.png)

lmunozm1702 replied:
Hi!! yes, but the instructions says "don't use javascript yet", and because I let "all opened" the lists! Thank's a lot!!

jerryowusu replied:
> Hi!! yes, but the instructions says "don't use javascript yet", and because I let "all opened" the lists! Thank's a lot!!

I undestand you perfectly well, @lmunozm1702. But it should be just like in the design, same design, same arrows, same icons, etc. No need to add anything extra. :stuck_out_tongue_winking_eye: Just like the image below. 

![Screenshot from 2022-09-30 06-41-23](https://user-images.githubusercontent.com/65233860/193207132-37a207cb-1a8b-4091-8303-94e97b1dd423.png)

jerryowusu reviewed (approved):
Hi @lmunozm1702 :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![good job](https://user-images.githubusercontent.com/65233860/193158929-68f528ac-2dfe-47a5-a238-0c5c2f0fde73.gif)

## Highlights
:heavy_check_mark: Followed GitHub Flow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: All Linters are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
